contrib/registry/servicecomb/registry.go

```diff
@@ -0,0 +1,171 @@
+package servicecomb
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/go-chassis/cari/discovery"
+	pb "github.com/go-chassis/cari/discovery"
+	"github.com/go-chassis/cari/pkg/errsvc"
+	"github.com/go-chassis/sc-client"
+	"github.com/go-kratos/kratos/v2/log"
+	"github.com/go-kratos/kratos/v2/registry"
+	"github.com/gofrs/uuid"
+	"os"
+	"time"
+)
```

Comment:
The import module is ordered as standard library, your project, 3rd.
```
import(
    "io"

    "my project"

    "3rd"
)
```

Reply:
done